cleanlab/object_detection/filter.py

```diff
@@ -195,8 +210,10 @@ def _find_label_issues_per_box(
         else:
             score_per_box[np.isnan(score_per_box)] = 1.0
             score_per_box = score_per_box
-            issue_per_box = score_per_box <= threshold
-            is_issue_per_box.append(issue_per_box)
+            issue_per_box = []
+            for i in range(len(score_per_box)):
```

Comment:
Is this equivalent to: `issue_per_box = score_per_box <= thr_classes[idx] * threshold_factor`

Reply:
we can't iterate here: thr_classes is a list[list]]
```
issue_per_box = []
            for i in range(len(score_per_box)):
                issue_per_box.append(score_per_box[i] <= thr_classes[idx][i] * threshold_factor)
```